recipes/quickfix/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+from conans import ConanFile, CMake, tools
+import shutil
+import os
+import re
+
+
+class QuickfixConan(ConanFile):
+    name = "quickfix"
+    license = "The QuickFIX Software License, Version 1.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://www.quickfixengine.org"
+    description = "QuickFIX is a free and open source implementation of the FIX protocol"
+    topics = ("conan", "QuickFIX", "FIX", "Financial Information Exchange", "libraries", "cpp")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"ssl": [True, False], "fPIC": [True, False]}
```

Comment:
quickfix can also be build with MySQL and PostgreSQL support: https://github.com/quickfix/quickfix/blob/6908dc8c1084eeb7af7f322d35216e9965684489/CMakeLists.txt#L23
A small patch should allow to build static lib on Linux/Macos (eventually raise ConanInvalidConfiguration if Visual Studio and shared, I don't know if symbols are properly exported): https://github.com/quickfix/quickfix/blob/6908dc8c1084eeb7af7f322d35216e9965684489/src/C%2B%2B/CMakeLists.txt#L65

Reply:
Yes, I thought of first getting this first PR approved before going on with including those options.

But if you think it's better I can add support to those now.

Reply:
let's keep it simple now and improve the recipe later in a different PR

Reply:
Hi, there, @SpaceIm and @danimtb . I managed to add Postgres support and also the option to build a shared library under non windows os's. I couldn't make MySQL compile as a dependency (I don't know why, just didn't pursue it after the first errors).

I pushed these changes into another branch: https://github.com/tarc/conan-center-index/tree/quickfix-static

I need to know if it's better to push them into this present PR or to open another one (given that people have already taken time revising this PR).